app/src/main/java/com/kks/nimblesurveyjetpackcompose/ui/presentation/survey/SurveyDetailScreen.kt

```diff
@@ -40,12 +43,15 @@ import com.kks.nimblesurveyjetpackcompose.ui.theme.NeuzeitFamily
 import com.kks.nimblesurveyjetpackcompose.ui.theme.White70
 import com.kks.nimblesurveyjetpackcompose.util.TWEEN_ANIM_TIME
 import com.ramcosta.composedestinations.annotation.Destination
+import com.ramcosta.composedestinations.navigation.DestinationsNavigator
+import com.ramcosta.composedestinations.navigation.EmptyDestinationsNavigator
 
 @OptIn(ExperimentalPagerApi::class)
 @Destination
 @Composable
-fun SurveyDetailScreen(survey: Survey) {
+fun SurveyDetailScreen(navigator: DestinationsNavigator, survey: Survey) {
     var currentPage by remember { mutableStateOf(0) }
+    val startSurveyDescription = stringResource(id = R.string.survey_detail_start_survey)
     val placeholderPainter = rememberAsyncImagePainter(model = survey.coverImagePlaceholderUrl)
 
     Box(modifier = Modifier.fillMaxSize()) {
```

Comment:
Have you considered using `Scaffold` instead of a `Box`? 🤔 Just wondering if that would help with proper toolbar alignment, it's still not aligned properly based on the figma design 👀 

Current:
<img width="144" alt="Screenshot 2565-09-02 at 17 45 30" src="https://user-images.githubusercontent.com/53168251/188123598-b04ea190-f9d6-4818-baab-8f9aed2bac73.png">

Figma:
<img width="144" alt="Screenshot 2565-09-02 at 17 46 02" src="https://user-images.githubusercontent.com/53168251/188123630-05732266-0325-4bd7-9638-03a1b5e9bd48.png">

Reply:
<img width="326" alt="Screen Shot 2022-09-02 at 6 07 14 PM" src="https://user-images.githubusercontent.com/32578035/188127098-eac6e26d-2049-4329-88c4-fdd0e56094a7.png">

@ryan-conway I have made changes in `Survey question screen UI` PR. 👍